phpunit/class-wp-theme-json-resolver-test.php

```diff
@@ -177,7 +177,7 @@ private static function recursive_ksort( &$array ) {
 	function test_merges_child_theme_json_into_parent_theme_json() {
 		switch_theme( 'block-theme-child' );
 
-		$actual_settings   = WP_Theme_JSON_Resolver_Gutenberg::get_theme_data()->get_settings();
+		$actual_settings   = WP_Theme_JSON_Resolver::get_theme_data()->get_settings();
```

Comment:
This line shouldn't be reverted, as the test should be for the Gutenberg version of the class.